openy.install

```diff
@@ -865,8 +865,8 @@ function openy_update_8057() {
   $query->addField('n', 'nid');
   $resultField = $query->execute()->fetchField();
   if ($resultField) {
-    $message = t('Visit <a href="/node/' . $resultField . '/edit">Locations edit page</a> and 
-      delete "Search by Amenities" Limited Time Offer paragraph in Header area. 
```

Comment:
This should not happen in our PRs

Reply:
@anprok please don't include changes not-related to the search functionality to our PRs (if it's not a blocker for you).